### PR TITLE
ASIC internal temperature sensors support

### PR DESCRIPTION
**Pull request by santhosh-kt**
ASIC internal temperature sensors support

More details:
Azure/sonic-swss#1517

Details if related
HLD Location: https://github.com/Azure/SONiC/blob/master/doc/asic_thermal_monitoring_hld.md

### CODE REVIEW

File: common/schema.h [hunk 1]
```diff
@@ -74,6 +74,9 @@ namespace swss {
 #define APP_STP_FASTAGEING_FLUSH_TABLE_NAME "STP_FASTAGEING_FLUSH_TABLE"
 #define APP_STP_BPDU_GUARD_TABLE_NAME       "STP_BPDU_GUARD_TABLE"
 
+/***** ASIC DATABASE *****/
+#define ASIC_SENSORS_TABLE_NAME	            "SENSORS"
```
Comment (jleveque) [inline common/schema.h:78]: Please left-align `"SENSORS"` with `"MUX_CABLE_TABLE"`

File: common/schema.h [hunk 2]
```diff
@@ -193,6 +196,8 @@ namespace swss {
 #define CFG_VLAN_INTF_TABLE_NAME          "VLAN_INTERFACE"
 #define CFG_VLAN_SUB_INTF_TABLE_NAME      "VLAN_SUB_INTERFACE"
 
+#define CFG_ASIC_SENSORS_TABLE_NAME	  "ASIC_SENSORS"
```
Comment (jleveque) [inline common/schema.h:199]: Please left-align `"ASIC_SENSORS"` with `"VLAN_SUB_INTERFACE"`

**Review by jleveque — CHANGES_REQUESTED** [review]

**Review by jleveque — APPROVED** [review]

File: common/schema.h [hunk 3]
```diff
@@ -74,6 +74,9 @@ namespace swss {
 #define APP_STP_FASTAGEING_FLUSH_TABLE_NAME "STP_FASTAGEING_FLUSH_TABLE"
 #define APP_STP_BPDU_GUARD_TABLE_NAME       "STP_BPDU_GUARD_TABLE"
 
+/***** ASIC DATABASE *****/
+#define ASIC_SENSORS_TABLE_NAME             "SENSORS"
```
Comment (qiluo-msft) [inline common/schema.h:78]: >ASIC_SENSORS_TABLE_NAME [](start = 8, length = 23)

This is not in design doc.

Comment (santhosh-kt) [reply common/schema.h:78]: My Apology.
I had two different clients with different table names(One for prototype and another for PR).
Didn't update the table name back.
Will update the thread.

Comment (santhosh-kt) [reply common/schema.h]: Now ASIC_TEMPERATURE_INFO_TABLE_NAME is matching with Azure/sonic-swss#1517

**Review by qiluo-msft — APPROVED** [review]

**Review by jleveque — APPROVED** [review]

**Comment by jleveque** [comment]
Submodule update PR: https://github.com/Azure/sonic-buildimage/pull/6042

File: common/schema.h [hunk 4]
```diff
@@ -74,6 +74,9 @@ namespace swss {
 #define APP_STP_FASTAGEING_FLUSH_TABLE_NAME "STP_FASTAGEING_FLUSH_TABLE"
 #define APP_STP_BPDU_GUARD_TABLE_NAME       "STP_BPDU_GUARD_TABLE"
 
+/***** ASIC DATABASE *****/
+#define ASIC_TEMPERATURE_INFO_TABLE_NAME    "ASIC_TEMPERATURE_INFO"
```
Comment (prsunny) [inline common/schema.h:78]: Sorry for the late comment - This should not belong here as the section is for APP DB entries. From the [usage](https://github.com/Azure/sonic-swss/blob/730e984e62b3bd47c22168e312c25bda2e850d28/orchagent/switchorch.cpp#L41), it looks to be part of STATE DB entries. 

Is it possible to correct this?

**Pull request merged**